app/assets/javascripts/admin/task/task_create_bulk_view.js

```diff
@@ -42,6 +42,7 @@ export type NewTask = {
   +taskTypeId: string,
   +csvFile?: File,
   +nmlFiles?: File,
+  +typ?: "skeleton" | "volume",
```

Comment:
From what I understand, this distinction is made in the task type, right? Probably forgot to remove it here?